components/centraldashboard/public/components/main-page.js

```diff
@@ -45,13 +48,82 @@ import {IFRAME_LINK_PREFIX} from './iframe-link.js';
 /**
  * Entry point for application UI.
  */
-export class MainPage extends utilitiesMixin(PolymerElement) {
+// eslint-disable-next-line max-len
+export class MainPage extends utilitiesMixin(mixinBehaviors([AppLocalizeBehavior], PolymerElement)) {
     static get template() {
         const vars = {logo};
         return html([
             `<style>${css.toString()}</style>${template(vars)}`]);
     }
 
+    constructor() {
+        super();
+        this.resources = {
```

Comment:
I'm guessing you're leaving these as they are until your mixin approach is reviewed. :+1: So just a reminder...